src/js/components/utils/deleteModal.tsx

```diff
@@ -39,6 +48,7 @@ const DeleteModal = ({
       deleteObject({
         objectType: modelType,
         object: model,
+        url: getModelDeleteUrl(modelType),
```

Comment:
I could go either way on whether the helper fn is useful here:

```suggestion
        url: modelType === OBJECT_TYPES.USER ? window.api_urls.current_user_detail() : undefined,
```